index.html

```diff
@@ -39,6 +39,80 @@ <h1>I'm José</br>
             </li>
         </ul>
     </section>
+    <section class="works">
+        <article>
+            <img src="images/Snapshoot Portfolio.svg">
+            <div>
+                <h2>Tonic</h2>
+                <div class="ref">
+                    <p class="category">Canopy</p>
+                    <p class="rest">Back End Dev</p>
+                    <p class="rest">2015</p>
+                </div>
+                <p class="copy">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="r1">
+                    <button class="a1">html</button>
+                    <button class="a1">css</button>
+                    <button class="a1">javascript</button>
+                </div>
+                <div class="action">
+                    <button class="a2">See project</button>
+                </div>
+            </div>
+        </article>
+
+    </section>
+    <section class="myself">
```

Comment:
Great job team 🤝

- [ ] Kindly make sure your design is similar to the design provided in the Figma template. As you can see there is a big difference between your design and the Figma design.
- Your design
![your about](https://user-images.githubusercontent.com/63917725/192119820-af923bc6-9aef-4ec2-a72c-9eb911a3721b.PNG)

- Figma design
![Figma about](https://user-images.githubusercontent.com/63917725/192119831-559f82b3-7ec2-4b8e-9a9d-0aaecebf872a.PNG)

Reply:
- [ ] Kindly make sure you implement the requested changes as suggested by the previous reviewer.  As you can see the template attached.